app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,17 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(self,
+                 nickname: str,
+                 musical_instrument: str,
+                 bow_level: int) -> None:

```

Comment:
Place each argument including `self` on a new line so that there would be no arguments on the same line with opening and closing parenthesis. It will help to improve the code style and readability. Apply this change here and in the other similar cases